checklist/checklist.csv/tests.csv

```diff
@@ -1,21 +1,25 @@
-5.6,Model Fitting,Check for Data Leakage,"Confirm that there is no leakage of data between training, validation, and testing sets, or across cross-validation folds, to ensure the integrity of the splits.","Data leakage can compromise the model's ability to generalize to unseen data, making it crucial to ensure datasets are properly segregated.",jordan2020
-6.1,Model Evaluation,Dummy Title,This is a dummy item and there is nothing needed to act on.,This is a dummy item to show how the checklist items would be stored inside this YAML file. It serves no other purpose.,"http://128.0.0.1, UBC-MDS"
-7.1,Artifact Testing,Invariance Tests,"There are tests to ensure making a set of preturbations to the input that is expected to cause no effect would not change the model's output. For example, in sentiment analysis tasks, changing the name of the subject or the name of a location should not affect the sentence's sentiment.",Models should be making predictions in a robust and consistent manner such that changes to the input which are known or expected to have no effect on the model's task should not affect the model's prediction.,"jordan2020, ribeiro2020accuracy"
-7.2,Artifact Testing,Directional Expectation Tests,"There are tests to ensure making a set of preturbations to the input that is expected to affect the predications in a particular direction would change the model's output in a same direction. For example, in a regression model, increasing the number of bathrooms (holding all other features constnat) should not cause a drop in price.",Models should be making predictions in a robust and consistent manner such that changes to the input which are known or expected to have an directional effect should be reflected in the model's predictions.,"jordan2020, ribeiro2020accuracy"
-7.3,Artifact Testing,Minimum Functionality Tests,"If there are critical scenarios where prediction errors lead to high consequences, tests should be written for ensuring the model's behavior in the scenario is expected. For example, in text classification, a test can check the model's output for sentences less than 5 words would perform as expected if the performance on short pieces of text is critical.","In real world, some model outcomes are often more important than others. Traditional Machine Learning systems optimize for overall quality which might lead to over-optimistic and/or non-representable performance metrics in cases where the importance of certain outcomes far outweigh the rest. For example, falsely identifying valid emails as spam has a more serious consequence than identifying a spam as a valid email.","jordan2020, ribeiro2020accuracy"
+ID,Topic,Title,Requirement,Explanation,References,is_evaluator_applicable

```

Comment:
Header
@jinyz8888 I think it is better to be `Is Evaluator Applicable` instead of `is_evaluator_applicable` as it looks better in PDF & other doc
<img width="630" alt="Screenshot 2024-05-27 at 2 18 52 PM" src="https://github.com/UBC-MDS/test-creation/assets/38127942/dfd4d8ab-0b47-4030-bebf-a53b16ed875f">
<img width="635" alt="Screenshot 2024-05-27 at 2 19 00 PM" src="https://github.com/UBC-MDS/test-creation/assets/38127942/e91cdd8a-adb1-4c7f-87e1-d766544624b9">

Reply:
@tonyshumlh are you sure we use `Is Evaluator Applicable` as the column name instead of `is_evaluator_applicable`?

Reply:
I have tested and it works well. 
We can change the name when other teammates / Simon & Tiffany has better suggestions